install.sh

```diff
@@ -302,13 +303,26 @@ nvm_do_install() {
     echo "   OR"
     echo "=> Append the following lines to the correct file yourself:"
     command printf "${SOURCE_STR}"
+    echo "=> Please also append the following lines to the if you are using bash/zsh shell:"
```

Comment:
Just realize that I can't do it since this message is for the users we can't detect their shell profile, if so, how could we know they are using bash or zsh I'll just drop this part now.